<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AGENDA</class>
 <widget class="QDialog" name="AGENDA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agenda</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15,15, 15);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_deletar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 0, 4);</string>
   </property>
   <property name="text">
    <string>DELETAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_voltar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0,0,245);</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_adicionar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>ADICIONAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_editar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>EDITAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Buscar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>BUSCAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
background-color: rgb(15, 15, 15);</string>
   </property>
   <property name="text">
    <string>Seus compromissos:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>461</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <column>
    <property name="text">
     <string>Titulo</string>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo</string>
    </property>
    <property name="textAlignment">
     <set>AlignJustify|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Button_reload">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RECARREGAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AGENDA</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AGENDA</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
